src/main/java/seedu/duke/logic/commands/favcommand/DeleteFavCommand.java

```diff
@@ -11,7 +11,7 @@
 
     public DeleteFavCommand(String index) throws CustomException {
         if (index.isEmpty() || index.isBlank()) {
```

Comment:
```suggestion
        if (index.isBlank()) {
```
check for whether index is blank is sufficient so perhaps u can omit the isEmpty check